webapp/src/main/java/org/opentestsystem/rdw/admin/security/session/SamlModule.java

```diff
@@ -0,0 +1,27 @@
+package org.opentestsystem.rdw.admin.security.session;
+
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import org.opentestsystem.rdw.admin.security.User;
+import org.springframework.security.providers.ExpiringUsernameAuthenticationToken;
+import org.springframework.security.saml.SAMLCredential;
+import org.springframework.security.saml.parser.SAMLCollection;
+import org.springframework.security.saml.parser.SAMLObject;
+
+/**
+ * This module registers de/serialization handlers for all classes
+ * serialized by SAML in the session store.
+ */
+public class SamlModule extends SimpleModule {
```

Comment:
some of this dup code going to find its way into common after the fix?

Reply:
I want to pull this into a web-base module once we move RDW_Admin into RDW_Reporting